maxim/logger/anthropic/__init__.py

```
@@ -1,5 +1,5 @@
 from .utils import AnthropicUtils
 from .client import MaximAnthropicClient
-from .async_client import MaximAnthropicAsyncClient
+from .utils import AnthropicUtils
```

Ruff (0.11.9)

Redefinition of unused `AnthropicUtils` from line 1

Remove definition: `AnthropicUtils`

(F811)